en-US/win10/samples/ZigBeeAdapterTutorial.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: default
+title: ZigBeeAdapterTutorial
+permalink: /en-US/win10/samples/ZigBeeAdapterTutorial.htm
+lang: en-US
+---
+
+# ZigBee sample
+Get the [code](https://github.com/ms-iot/samples/blob/develop/AllJoyn/AllJoynZigBeeAdapter/ZigBeeAdapter.zip?raw=true) on Github
+
+This document describes the setup of the ZigBee adapter for Device System Bridge (DSB) on Windows 10. When using it you will be able to expose ZigBee devices to AllJoyn.
+
+## What is ZigBee?
+
+ZigBee is a low cost, low power, wireless communications standard designed to allow devices to communicate with another. In addition to the communication protocol, ZigBee standard also defines profiles, such as ZigBee Light Link or Home Automation, which themselves define device. For example, ZigBee Light Link will define what a light is, what a dimmable light is, and so on. Each device uses clusters defined in the ZigBee Cluster Library (ZCL) to specify what they can do, what they expose…
+
+See [ZigBee standard](http://www.zigbee.org) for more information about ZigBee, ZigBee profiles, ZCL clusters...
+
+Acronyms:
+- ZDO: ZigBee Device Object
+- ZCL: ZigBee Cluster Library
+
+## Prerequisites
+1. XBee ZigBee module from [Digi](http://www.digi.com), e.g.: XB24 Z7PIT-004
+2. XBee Explorer USB dongle from [SparkFun](https://www.sparkfun.com)
```

Comment:
Why not link to the actual XBee Explorer Dongle rather than the generic Sparkfun site?
https://www.sparkfun.com/products/11697

Or was the goal to have people look through each linked site to have to buy / find out where to buy the parts?